src/main/java/bridge/domain/Player.java

```diff
@@ -0,0 +1,37 @@
+package bridge.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Player {
+    public static final String MOVABLE = "UD";
+    private List<String> record = new ArrayList<>();
+    int tryNumber;
```

Comment:
```suggestion
private static final String MOVABLE = "UD";

private final List<String> records = new ArrayList<>();
private int tryNumber;
```
필드의 순서도 컨벤션입니다!
tryNumber의 접근 제어자는 default 인데요, private과 어떤 차이가 있을까요??
컬렉션의 경우는 보통 복수형으로 써주는 것 같아요